extensions/wikibase/tests/src/org/openrefine/wikibase/commands/LoginCommandTest.java

```diff
@@ -97,38 +97,38 @@ public void setUp() throws Exception {
     public void testNoApiEndpointPost() throws ServletException, IOException {
         when(request.getParameter("csrf_token")).thenReturn(Command.csrfFactory.getFreshToken());
         command.doPost(request, response);
-        assertEqualAsJson("{\"code\":\"error\",\"message\":\"missing parameter 'wb-api-endpoint'\"}", writer.toString());
+        assertEqualsAsJson("{\"code\":\"error\",\"message\":\"missing parameter 'wb-api-endpoint'\"}", writer.toString());
```

Comment:
```suggestion
        assertEqualsAsJson(writer.toString(), "{\"code\":\"error\",\"message\":\"missing parameter 'wb-api-endpoint'\"}");
```

The parameter order is different between the two versions of the functions, so these need to be swapped. Ditto for all the occurrences.

Reply:
It's a change that should be relatively easy to do with the "Inlining" refactoring method of IDEs like Eclipse or IntelliJ by the way.

Reply:
Ah, I thought the refactoring took care of that but of course the tests pass anyway! Nice catch.